packages/flutter/lib/src/material/page.dart

```diff
@@ -39,9 +39,12 @@ class _MaterialPageTransition extends AnimatedWidget {
   }
 }
 
-const Duration kMaterialPageRouteTransitionDuration = const Duration(milliseconds: 150);
-
+/// A modal route that replaces the entire screen with a material design transition.
```

Comment:
we should say what kind of transition (fading in while sliding up).

(I plan to add another MaterialPageRoute-style class that does a circle wipe transition.)